source/features/link-to-file-in-file-history.tsx

```diff
@@ -40,5 +32,9 @@ features.add({
 	include: [
 		pageDetect.isRepoCommitList
 	],
+	exclude: [
+		// Probably looking at the base /commits/<branch> page, not a subfolder or file.
+		() => !select.exists('.breadcrumb')
```

Comment:
I think this can now be determined via parseRoute. “If filePath === '', then this feature, like others, shouldn’t run”